README.md

```diff
@@ -37,9 +37,27 @@ ros2 run topic_tools relay base_scan my_base_scan
 - `lazy` (bool, default=False)
     - If True, only subscribe to `input_topic` if there is at least one subscriber on the `output_topic`
 
+### RelayField
+
+RelayField is a ROS 2 node that allows to republish data in a different message type
+
+#### Usage
+
+```
+ros2 run topic_tools transform <input topic> <output topic> <output type> [<expression on m>] [--import <modules>] [--field <topic_field>]
```

Comment:
Something is wrong because transform is being used incorrectly.

Reply:
can you open a new issue (or a PR)?

Reply:
Done. #77

Reply:
Oops, you're right. The changes in #77 look good to me. Thanks!